client/functions.lua

```diff
@@ -250,61 +244,40 @@ function QBCore.Functions.IsWearingGloves()
 end
 
 function QBCore.Functions.GetClosestPlayer(coords)
-    local ped = PlayerPedId()
-    if coords then
-        coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords
-    else
-        coords = GetEntityCoords(ped)
-    end
-
+    coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords or GetEntityCoords(cache.ped)
```

Comment:
This line is repeated several times. Consider extracting to a local function.

Reply:
I probably wont make this into a local function would be kinda wierd, the only reason why i reused the code is because it just replaced the 5-6 long lines of wierdness that was going on last time.

Reply:
Not sure why it would be weird to extract duplicate code into a function, but up to you.